components/automate-ui/src/app/components/calendar/calendar.component.ts

```diff
@@ -45,7 +45,7 @@ export class CalendarComponent {
 
   @Input()
   set selected(input) {
-    const date = m.isMoment(input) ? input : m(input);
+    const date = m.isMoment(input) ? input : m.utc(input);
```

Comment:
I'm thinking this needs to be altered to check not only if `input` is a moment object, but also if it's using utc?

Reply:
@msorens do you see any large issue with changing this to be

`const date = m.utc(input)`

so that we can be absolutely sure it's in utc?

per https://github.com/moment/moment/issues/793 , there doesn't seem to be a rock solid way of easily checking if the object is already utc.  The suggestion from moment team mentioned "only for debugging"

Reply:
or possibly check using `isSame()` https://momentjs.com/docs/#/query/is-same/ , however i'm confused by the note at the bottom of the isSame() documenation as to whether they are saying do not use isSame() with nothing passed in, or if they've discovered it is not reliable....
>NOTE: moment().isSame() has undefined behavior and should not be used! If the code runs fast the initial created moment would be the same as the one created in isSame to perform the check, so the result would be true. But if the code runs slower it's possible that the moment created in isSame is measurably after the one created in moment(), so the call would return false.

Reply:
a date that has been acted on by `moment()` is converted into an object of key value pairs.  one of those keys is an `_isUTC` boolean which by default is set to False.  To answer your question, when an input is already a moment, then you do a `utc()` on it, the utc flag is then set to True.  Nothing else in the object is affected.  The downside being that it runs a function that would be unnecessary if it was already utc.